python/examples/tools/base.py

```diff
@@ -0,0 +1,13 @@
+import asyncio
+
+from beeai_framework.tools.weather.openmeteo import OpenMeteoTool, OpenMeteoToolInput
+
+
+async def main() -> None:
+    tool = OpenMeteoTool()
+    result = tool.run(input=OpenMeteoToolInput(location_name="New York"))
```

Comment:
is there a reason you left out `start_date` and `end_date` inputs? They're in the TS implementation

Reply:
No strong reason other than this being a base example, added for next commit.